.env.testnet

```diff
@@ -0,0 +1,8 @@
+COMPOSE_PROJECT_NAME="dash_masternode_testnet"
+PRESET=testnet
+CORE_VERSION=latest
```

Comment:
Not sure is good idea to use latest. Using specific version we can force to update + more reliable coz we control it. In other presets we set 0.15

Reply:
I set this to latest because codablock usually runs binaries from the core `develop` branch in his testnet containers, since this is what is being tested.